lib/mayaUsd/render/vp2RenderDelegate/basisCurves.cpp

```diff
@@ -71,10 +71,6 @@ struct CommitState
 
     //! If valid, new index buffer data to commit
     int* _indexBufferData { nullptr };
-    //! If valid, new color buffer data to commit
```

Comment:
The changes to basisCurves are mirroring part of the refactor in did in mesh. It's all centered around using _CommitVertexBuffer to create a small main thread task that commits one buffer, vs. saving all the committing for later.

In basisCurves.cpp I didn't go as far with the refactor as I did in mesh.cpp, but I believe the same types of changes I was making there could be made here.

Reply:
Ok. And I assume there won't be a bigger difference which would make it more difficult for us to unify the Rprim logic (such as a base class for HdVP2Rprim) after this change?

Reply:
I think that this change doesn't make that refactor any more complicated.